UnityProject/Assets/Scripts/UI/Systems/MainHUD/StatsTab.cs

```diff
@@ -14,13 +14,25 @@ public class StatsTab : SingletonManager<StatsTab>
 
 	void OnEnable()
 	{
+		Invoke("SetScrollToTop",0.1f);
```

Comment:
can you remove this, since its unneeded?

Reply:
Dunno why you needed this specifically here, but ok